packages/lit-dev-content/site/tutorials/content/svg-templates/00.md

```diff
@@ -0,0 +1,14 @@
+SVG enables artists to create scalable graphics through an XML-like syntax. It's supported in most modern browsers.
```

Comment:
probably all modern browsers

Reply:
sg :)
changed it to "Both are supported by all modern browsers."
both meaning lit and svg